### DISCUSSION TITLE
Compatibilty questions

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#904] [General]
Is ArchR now compatible with R version 4.0?

**Comment by rcorces** [comment]
yes

**Comment by sylestiel** [comment]
awesome thanks!

**Comment by NoemieL** [comment]
Hi, Is it also compatible with R v 4.1.1? I get the same warning as in #357 without any info in the log file. 
If it is not, is the version 4.1.0 compatible?
Thanks

**Reply by rcorces** [reply]
I havent specifically tested it on 4.1.1
The error you are referencing is a bit non-descript. If you are using `release_1.0.2` try reinstalling to make sure you have the most recent update because there was a critical error in arrow file creation that has now been addressed.

**Reply by NoemieL** [reply]
I have the master version 1.0.1 which was installed this weekend. Do you recommend using another release?

**Reply by rcorces** [reply]
no 1.0.1 is stable and fine.
You're saying that the tutorial code on the tutorial dataset does not work for you on 4.1.1 but it was working previously on another R version?

**Reply by NoemieL** [reply]
My own data were working well with R version 3.6.3 and the ArchR master. We recently change for version 4.1.1 and now I have the warnings mention previously and no ArrowFiles for the second step addDoubletScores().

**Reply by rcorces** [reply]
Please test on the tutorial data and let me know

**Reply by NoemieL** [reply]
I try with the tutorial. I have the same problem. I get the fragment pdfs but the TSS pdfs are empty.
[ArchR-createArrows-56b7741e1f3ac-Date-2021-09-13_Time-22-45-32.log](https://github.com/GreenleafLab/ArchR/files/7157822/ArchR-createArrows-56b7741e1f3ac-Date-2021-09-13_Time-22-45-32.log)

ArrowFiles <- createArrowFiles(
  inputFiles = inputFiles,
  sampleNames = names(inputFiles),
  minTSS = 4,
  minFrags = 1000, 
  addTileMat = TRUE,
  addGeneScoreMat = TRUE
)

Using GeneAnnotation set by addArchRGenome(Hg19)!
Using GeneAnnotation set by addArchRGenome(Hg19)!
ArchR logging to : ArchRLogs/ArchR-createArrows-56b7741e1f3ac-Date-2021-09-13_Time-22-45-32.log
If there is an issue, please report to github with logFile!
2021-09-13 22:45:33 : Batch Execution w/ safelapply!, 0 mins elapsed.
ArchR logging successful to : ArchRLogs/ArchR-createArrows-56b7741e1f3ac-Date-2021-09-13_Time-22-45-32.log
Warning message:
In mccollect(jobs) : 3 parallel jobs did not deliver results

**Reply by rcorces** [reply]
Ok. But your log file shows that Arrow file creation was successful. So maybe there is a problem with plotting but you are not getting an error.

**Reply by NoemieL** [reply]
Yes sorry Arrow files are created but the Arrowfiles object needed for the next step addDoubletScores() is not generated. Do you recommend to use another version of R? if yes which one?

**Reply by rcorces** [reply]
Many users are successfully using 4.0.*
You can poke through questions and issue reports - the log files have the version numbers

**Reply by NoemieL** [reply]
With the R 4.1.0 seems to be ok.

**Reply by rcorces** [reply]
Thanks for letting us know. That is very bizarre. I will try to test on 4.1.1

**Reply by rcorces** [reply]
FYI, other users have also reported this and one potential fix is listed here:
https://github.com/GreenleafLab/ArchR/discussions/1044#discussioncomment-1405648

> I installed and read in the following to be able to run ArchR using R 4.1.1
library(parallel)
library(magick)

**Reply by rcorces** [reply]
fixed in code base here - https://github.com/GreenleafLab/ArchR/discussions/1044#discussioncomment-1654733
